aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -160,6 +161,27 @@ module aptos_framework::genesis {
         stake::on_new_epoch();
     }
 
+    // Creates the initial list of accounts and funds them with their initial balances.
+    public fun create_initial_accounts(
```

Comment:
Can you make this function private to ensure it can't be called outside of the genesis flow?